File: Java/PB with Java/FirstStepsInProgr/src/Projects.java
import java.util.Scanner;

public class Projects {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter name: ");
        String firstName = scanner.nextLine();
        System.out.print("Enter projects: ");
        int projects = Integer.parseInt(scanner.nextLine());
        int hours = 3 * projects;

        System.out.printf("\"The architect %s will need %d hours to complete %d project/s.",
                firstName, hours, projects);
    }
}

File: C++/Programming Basics with C++/First Steps In Programming - Lab and Exc/Projects/Projects.cpp
#include <iostream>
#include <string>

int main() {
    std::string firstName;
    int projects;

    std::cout << "Enter name: ";
    std::getline(std::cin, firstName);
    std::cout << "Enter projects: ";
    std::cin >> projects;

    int hours = 3 * projects;

    std::cout << "\"The architect " << firstName << " will need " << hours << " hours to complete " << projects << " project/s." << std::endl;

    return 0;
}
